MobileCenterCrashes/MobileCenterCrashesTests/MSHandledErrorLogTests.m

```diff
@@ -0,0 +1,125 @@
+#import "MSHandledErrorLog.h"
+#import "MSCrashesTestUtil.h"
+#import "MSException.h"
+#import "MSTestFrameworks.h"
+
+@interface MSHandledErrorLogTests : XCTestCase
+
+@property(nonatomic) MSHandledErrorLog *sut;
+
+@end
+
+@implementation MSHandledErrorLogTests
+
+#pragma mark - Housekeeping
+
+- (void)setUp {
+    [super setUp];
+    
```

Comment:
No empty line. We have a same rule for line break. Please remove all unnecessary empty lines in the PR.